### PR TITLE
Add changes to podman oci mount

### PR DESCRIPTION
**Pull request by enriquebelarte**
To be consistent with changes at this [PR](https://github.com/containers/ai-lab-recipes/pull/617) the usage of podman in this line needs to be changed accordingly.

/cc @Gregory-Pereira @rhatdan

### CODE REVIEW

File: training/nvidia-bootc/Containerfile [hunk 1]
```diff
@@ -178,7 +178,7 @@ RUN for i in /usr/local/bin/ilab*; do \
 # Added for running as an OCI Container to prevent Overlay on Overlay issues.
 VOLUME /var/lib/containers
 
-RUN if [ -f "/run/.input/ilab" ]; then \
```
Comment (rhatdan) [inline training/nvidia-bootc/Containerfile]: Should check
 /run/.input/instructlab-nvidia

Comment (rhatdan) [reply training/nvidia-bootc/Containerfile]: Could you do the same with intel-bootc and amd-bootc?

Comment (enriquebelarte) [reply training/nvidia-bootc/Containerfile]: Sure. Thank you Dan.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**